src/kit/internal/types.ts

```diff
@@ -7,6 +7,10 @@ export type RecordKey = string | number | symbol;
 export type NullOrUndefined<T = undefined> = T | null | undefined;
 export type Range<T = Primitive> = [T, T];
 
+// DEPRECATED
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export type RawJson = Record<string, any>;
+
```

Comment:
fix: think you removed where this was used, so it should be deleted

Reply:
I guess I removed a redundant one, but looks like this actually is still being used `DataGrid/columns.ts`